### PR TITLE
feat(ats): add beta 3.0.0 readme

### PR DESCRIPTION
**Pull request by jay-pulsifi**

### CODE REVIEW

**Comment by davidhongpulsifi** [comment]
- Additional scores 
Missing Score type for reasoning overall?

File: beta/3.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+  - **Nullable**: Yes
+
+  - Each additional score object should contain:
+
+    - **`score_format`**: The format of the score (e.g., 100 for percentage).
+
+      - **Type**: Integer
+      - **Nullable**: No
+
+    - **`score_type`**: The type of the score (e.g., `work_value`, `work_style`,`work_experience`,`reasoning_verbal`, `reasoning_logical`,`reasoning_numeric`, `interest_riasec`, `hard_skills`).
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md]: if this come from ta-report-fn,  we have this naming now

https://github.com/Pulsifi/talent-acquisition-report-fn/blob/a3e003072006214a584e8df54e9dba88ca819b3d/src/interface/assessment-report-request.ts#L116-L117

File: beta/3.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+- **`candidate_id`**: The unique identifier for the candidate in the Pulsifi system.
+  - **Required**: Yes
+  - **Type**: String
+  - **Max Length**: Varies
+  - **Nullable**: No
+
+#### **Example cURL:**
+
+```bash
+  curl -X GET 'https://api.pulsifi.me/partner/v1.0/standard/candidate/{candidate_id}/details' \
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md]: plural 

/candidates

remove /details

File: beta/3.0.0/README.md [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+
+---
+
+## **Endpoints**
+
+### **1. Create Candidate Invitation**
+
+Creates a new candidate invitation in the Pulsifi system.
+
+- **Endpoint:** `POST /partner/v1.0/standard/candidate`
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md]: plural

/candidates

File: beta/3.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+- **`deadline`**: The assessment invitation deadline.
+  - **Required**: No
+  - **Type**: String (UTC Date)
+  - **Nullable**: No
+  - **Description**: Must be within 3 months from the current date.
+
+#### **Example cURL:**
+
+```bash
+curl -X POST 'https://api.pulsifi.me/partner/v1.0/standard/candidate' \
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md]: plural 
/candidates

File: beta/3.0.0/README.md [hunk 5]
```diff
@@ -0,0 +1,447 @@
...
+  - **Type**: String (UTC Date)
+  - **Nullable**: No
+
+---
+
+### **2. Get Candidate Result / Details**
+
+Use this endpoint to get candidate results or details.
+
+- **Endpoint:** `GET /partner/v1.0/standard/candidate/{candidate_id}`
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md]: plural


/candidates

File: beta/3.0.0/README.md [hunk 6]
```diff
@@ -0,0 +1,447 @@
...
+    "last_name": "Doe",
+    "deadline": "2021-08-12T12:21:59Z"
+  }'
+```
+
+#### **Response Example:**
+
+```json
+{
+  "status": "invited",
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md:228]: make response comply with payload like
- email
- first_name
- last_name
- is_anonymous_candidate
- skills
- work_experiences

File: beta/3.0.0/README.md [hunk 7]
```diff
@@ -0,0 +1,447 @@
...
+- **`first_name`**: The candidate's first name.
+
+  - **Required**: No (if `is_anonymous_candidate` is `true`)
+  - **Type**: String
+  - **Max Length**: 255 characters
+  - **Nullable**: Yes
+
+- **`last_name`**: The candidate's last name.
+
+  - **Required**: No (if `is_anonymous_candidate` is `true`)
```
Comment (davidhongpulsifi) [inline beta/3.0.0/README.md:134]: Can make this as not required and default to false , move it to last field 

See if we can offload anonymous mode to client 

Like Testlify or willo  also wont have this 

It up us to send in a fake email / name or not

Comment (davidhongpulsifi) [reply beta/3.0.0/README.md:134]: Same behavior as deadline right? Not required and default to 3 months

Comment (jay-pulsifi) [reply beta/3.0.0/README.md:134]: Aite make sense.

Make email , first_name, last_name (optional)

We default them with fake email/name on our side if not provide.
Remove is_anonymous key.

Yeah deadline will be default 3 months if not provided.

Comment (davidhongpulsifi) [reply beta/3.0.0/README.md:134]: as per discussion
- stick with is_anonymous , not required,  default false **Don sell this in api doc in future

**Pull request merged**